src/components/text-input/text-input.stories.tsx

```diff
@@ -56,9 +56,9 @@ export const withIconAndComplexHelp = () => (
     label="Hand size"
     placeholder="Type here..."
     help={
-      <Paragraph paragraphSize="small" marginTop="8px" color="grey.light.1">
+      <Paragraph paragraphSize="small" marginTop="8px" color="neutral.2">
```

Comment:
```suggestion
      <Paragraph paragraphSize="small" marginTop="8px" color="neutral.4">
```
![](https://media.giphy.com/media/ZNMa7ocjGc0PTlHrP4/giphy.gif)